hybrid/flow.py

```diff
@@ -163,6 +172,104 @@ def halt(self):
             component.stop()
 
 
+class Branches(Runnable, traits.MIMO):
+    """Parallelly executed :class:`~hybrid.core.Runnable` components.
+
+    Args:
+        *branches ([:class:`~hybrid.core.Runnable`]):
+            Runnable branches listed as positional arguments.
+
+    Input:
+        :class:`~hybrid.core.States`
+
+    Output:
+        :class:`~hybrid.core.States`
+
+    Note:
+        `Branches` is also available via implicit parallelization binary
+        operator `&`.
+
+    Examples:
+        This example runs two branches, a classical tabu search and a random
+        sampler, until both terminate.
+
+        >>> Branches(TabuSubproblemSampler(), RandomSubproblemSampler())    # doctest: +SKIP
+
+        Alternatively:
+
+        >>> TabuSubproblemSampler() & RandomSubproblemSampler()     # doctest: +SKIP
+
+    """
+
+    def __init__(self, *branches, **runopts):
+        super(Branches, self).__init__(**runopts)
+        self.branches = tuple(branches)
```

Comment:
`*args` maps to a tuple by default

Reply:
You're right, I tupleized out of habit. :)